velox/docs/functions/spark/datetime.rst

```diff
@@ -7,6 +7,12 @@ Convenience Extraction Functions
 
 These functions support TIMESTAMP and DATE input types.
 
+.. spark:function:: date_sub(start_date, num_days) -> date
+
+    Returns the date that is num_days before start_date.
+
+    num_days can be positive or negative.
```

Comment:
Would you document the overflow / wrap-around behavior to avoid surprising users?